SpatialGDK/Extras/internal-documentation/release-process.md

```diff
@@ -65,38 +65,17 @@ It is vital that you test using the docs for the release version that you are ab
 
 If at any point in the below validation steps you encounter a blocker, you must fix that defect prior to releasing.
 
-There are two ways to do this, you must decide on the most appropriate one.
-* Delete the existing release branches and branch new ones from the HEAD of `master` (in the case of the GDK and tutorials) and from the HEAD of `4.xx-SpatialOSUnrealGDK` (in `improbableio/UnrealEngine`).
-* `git cherry-[commit-hash]` the change into your release candidate branch. Be sure to cherry-pick any changes that your cherry-pick depends on. GDK changes often depend on engine changes, for example.
+The workflow for this is:
```

Comment:
I can't comment because of GitHub UI but:
Line 45:
1. `git clone` the [UnrealGDKThirdPersonShooter](https://github.com/spatialos/UnrealGDKTestSuite).
Should be:
1. `git clone` the [UnrealGDKTestSuite](https://github.com/spatialos/UnrealGDKTestSuite).

Reply:
I thought we're getting rid of TestSuite?

Reply:
@joshuahuburn good spot

@Vatyx we are but until that happens this is the correct process

Reply:
fixed